e2e/server_test.go

```diff
@@ -87,17 +87,17 @@ func TestProxyClient(t *testing.T) {
 	var testPreimage = []byte("inter-subjective and not objective!")
 
 	t.Log("Setting input data on proxy server...")
-	blobInfo, err := daClient.SetData(ts.Ctx, testPreimage)
+	blobInfo, err := daClient.SetData(ts.Ctx, testPreimage, server.DefaultCommitmentMode)
```

Comment:
knit - having commitment mode as part of the server package could result in ambiguity when imported in rollup repos